mobile/src/store/middlewares/handle-error/handle-error.middleware.ts

```diff
@@ -12,7 +12,10 @@ const handleError: Middleware =
   ({ dispatch }: HandleErrorParams) =>
   (next) =>
   (action) => {
-    if (action.error) {
+    if (
+      action.error &&
+      action.error.name !== CustomExceptionName.INVALID_CREDENTIALS
```

Comment:
You need to do sign out in case of error.name === CustomExceptionName.INVALID_CREDENTIALS